language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -139,15 +207,15 @@ instance SpendBudget (CekM uni) (WithMemory Term uni) where
                 when (exceedsBudget resb newBudget) $
                     throwingWithCause _EvaluationError
                         (UserEvaluationError $ CekOutOfExError resb newBudget)
-                        (Just term)
+                        Nothing  -- No value available for error
```

Comment:
It's not that it's super useful to return the term, evaluation of which consumed the last bits of gas. What people are normally interested in is where most of it was spent (and how much more needed to finish the computation, but that is something that one can easily figure out by running the script locally).